src/Dfe.PlanTech.Web/TagHelpers/WarningComponentTagHelper.cs

```diff
@@ -16,14 +16,14 @@ public WarningComponentTagHelper()
     public override async Task ProcessAsync(TagHelperContext context, TagHelperOutput output)
     {
         var inner = await output.GetChildContentAsync();
-        var html = GetHtml(inner.GetContent());
+        var html = WarningComponentTagHelper.GetHtml(inner.GetContent());
```

Comment:
This was a sonar cloud complaint?? I would have thought its redundant to add this 🤔 harmless I suppose

Reply:
Not this specifically, but making the method static. I've removed the redundant class on this line now though